docs/modules/ROOT/pages/explanations/decisions/postgresql.adoc

```diff
@@ -0,0 +1,84 @@
+= PostgreSQL Operators
+
+== Problem
+
+Creating our own operator to deploy and manage PostgreSQL proved to be a gargantuan effort. In order to keep in line with the newly specified https://kb.vshn.ch/app-catalog/reference/framework-requirements.html#_for_service_maintainer["more buy than make"] approach, we're evaluating the viability of using existing operators for the use of converged services.
+
+== Evaluated Operators
+
+[cols="1,1,1,1,1,1,1"]
+|===
+|Requirements |Percona |Stackgres |CloudnativePG |Zalando |Crunchydata |Bitnami Helmchart
+|Superuser Access |✅ |✅ |✅ |✅ |✅ |✅
+
+|Database Management |❌ |❌ |❌ |✅ |✅ |❌
+
+|User Management |❌ |❌ |❌ |✅ |✅ |✅
+
+|Service Metrics |❌ |✅ |✅ |❌ |✅ |✅
+
+|Maintenance Schedule |✅ |✅ |❌ |❌ |❌ |❌
+
+|Backup Schedule |✅  |✅  |✅  |✅  |✅  |❌
+
+|Self Service Backup Restore |✅ |✅ |✅ |✅ |✅ |❌
+
+|Encryption at rest |✅ |✅ |✅ |✅ |✅ |✅
+
+|Connection limit of 25/Gb Memory |✅ |✅ |✅ |❌ |✅ |✅
+
+|Extension Management |❌ |✅ |❌ |✅ |❌ |✅
+
+|Custom PostgreSQL Settings |✅ |✅ |✅ |❌ |✅ |✅
+
+|In-Place Major Upgrade |✅ |✅ |❌ |❌ |❌ |❌
+
+|License |Apache License 2.0 |AGPL-3.0 |Apache License 2.0 |MIT |Apache License 2.0 | Apache License 2.0
+
+|Upstream Support |Some support available not clear if for operators |✅ |✅ (via EDB) |❌ |✅ |❌
+|===
+
+https://wiki.vshn.net/display/VHT/PostgreSQL+Operator+Evaluations[More detailed] comparison is in our internal wiki.
```

Comment:
Please do not link to our wiki, that's not useful for external readers. I'd extract the most important parts of the wiki page and when this page is released, delete the wiki page. Otherwise, I'm sure this would cause friction.

Reply:
I also favor to integrate the wiki page here.